zuul.d/adoption.yaml

```diff
@@ -95,9 +95,14 @@
                 ip: 172.20.0.100
                 config_nm: false
 
+# TODO(marios): remove this once we finish rename OSPRH-8452
 - job:
     name: cifmw-data-plane-adoption-osp-17-to-extracted-crc
-    parent: data-plane-adoption-osp-17-to-extracted-crc
+    parent: adoption-standalone-to-crc-ceph-provider
```

Comment:
TODO: add description (via Lewis/reviews call)

Reply:
Ack, I'm happy with this once otherwise